src/components/Link.tsx

```diff
@@ -120,6 +118,32 @@ export const BaseLink = forwardRef(function Link(
     )
   }
 
+  if (isInternalPdf) {
+    return (
+      <ChakraLink
+        isExternal
```

Comment:
this should be removed?

Reply:
The PDF should be opened in a separate tab, which means this is needed.

Reply:
Correct. We force it to open in a new tab.